src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -604,44 +604,124 @@ internal BoundStatement BindLocalDeclarationStatement(LocalDeclarationStatementS
 
         private BoundStatement BindDeclarationStatementParts(LocalDeclarationStatementSyntax node, DiagnosticBag diagnostics)
         {
-            var typeSyntax = node.Declaration.Type.SkipRef(out _);
+            var variableList = node.Declaration.Variables;
+            int variableCount = variableList.Count;
+
             bool isConst = node.IsConst;
+            bool IsUsing = node.UsingKeyword != default;
+            bool IsMultiple = variableCount > 1;
 
+            var typeSyntax = node.Declaration.Type.SkipRef(out _);
             bool isVar;
             AliasSymbol alias;
             TypeSymbol declType = BindVariableType(node.Declaration, diagnostics, typeSyntax, ref isConst, isVar: out isVar, alias: out alias);
 
-            // UNDONE: "possible expression" feature for IDE
+            Conversion iDisposableConversion = default;
+            MethodSymbol disposeMethod = default;
 
             LocalDeclarationKind kind = LocalDeclarationKind.RegularVariable;
             if (isConst)
             {
                 kind = LocalDeclarationKind.Constant;
             }
-            else if (node.UsingKeyword != default)
+            if (IsUsing)
             {
                 kind = LocalDeclarationKind.UsingVariable;
+                return BindUsingVariableDeclaration(
+                                                    this,
+                                                    diagnostics,
+                                                    diagnostics.HasAnyErrors(),
+                                                    node,
+                                                    node.Declaration,
+                                                    out iDisposableConversion,
+                                                    out disposeMethod);
+                
+            }
+            else
+            {
+                if (IsMultiple)
```

Comment:
Is there a reason to change this from the original?
```
if (variableCount == 1)
{
    return BindVariableDeclaration(...);
}
else
{
    ...
    return new BoundMultipleLocalDeclarations(...);
}
```
If not, consider reverting to avoid unnecessary changes.